app/views/foreman/job_templates/change_content_source.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Change content source
+job_category: Katello
+model: JobTemplate
+provider_type: SSH
+description_format: Configure subscription manager to new content source (smart proxy)
+feature: katello_change_content_source
+template_inputs:
+- name: smart_proxy
+  required: true
+  input_type: user
+  value_type: resource
+  resource_type: SmartProxy
+  description: New content source (Smart Proxy)
```

Comment:
This "Smart Proxy" may be hard to translate to Capsule, if possible, just avoid that string here.